tests/components/ecobee/test_climate.py

```diff
@@ -331,3 +359,38 @@ async def test_set_fan_mode_auto(thermostat, data):
     data.ecobee.set_fan_mode.assert_has_calls(
         [mock.call(1, "auto", "nextTransition", holdHours=None)]
     )
+
+
+async def test_set_ventilator_mode_on(thermostat, data):
+    """Test set ventilator mode to on."""
+    data.reset_mock()
+    thermostat.set_ventilator_mode("on")
+    data.ecobee.set_vent_mode.assert_has_calls([mock.call(1, "on")])
+
+
+async def test_set_ventilator_min_on_time(thermostat, data):
+    """Test set ventilator mode to on."""
```

Comment:
Docstring for this, and the following, test functions look like they need to be updated.

Reply:
Good catch, I missed these. Will fix them.